pkg/actions/addon/create.go

```diff
@@ -40,6 +40,14 @@ func (a *Manager) Create(addon *api.Addon) error {
 	logger.Debug("addon: %v", addon)
 	namespace, serviceAccount := a.getKnownServiceAccountLocation(addon)
 
+	if addon.Tags != nil && len(addon.Tags) > 0 {
+		logger.Info("using provided Tags %q", addon.Tags)
```

Comment:
We don't need to log all tags here. Additionally, this will be printed for every addon using tags which can clutter the logs, so I think it's fine to not log anything.